GI01SUMO/GI01SUMO_R00006_ingest.csv

```diff
@@ -0,0 +1,76 @@
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl11/mopak/*.mopak.log,GI01SUMO-SBD11-01-MOPAK0000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl11/hyd1/*hyd1.log,GI01SUMO-SBD11-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl11/dosta1/*.dosta1.log,GI01SUMO-SBD11-04-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl11/spkir1/*.spkir1.log,GI01SUMO-SBD11-05-SPKIRB000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl11/metbk1/*.metbk1.log,GI01SUMO-SBD11-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.suna.suna_dcl_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl11/nutnr1/*.nutnr1.log,GI01SUMO-SBD11-08-NUTNRB000,recovered_host,Available,
+mi.dataset.driver.suna.suna_instrument_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/instruments/dcl11/NUTNR_sn*/DAT/*.CSV,GI01SUMO-SBD11-08-NUTNRB000,recovered_inst,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl12/syslog/*syslog.log,GI01SUMO-SBD12-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl12/optaa1/*.optaa1.log,GI01SUMO-SBD12-01-OPTAAD000,recovered_host,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl12/flort1/*.flort1.log,GI01SUMO-SBD12-02-FLORTD000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl12/hyd2/*hyd2.log,GI01SUMO-SBD12-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl12/pco2a/*.pco2a.log,GI01SUMO-SBD12-04-PCO2AA000,recovered_host,Available,
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl12/wavss/*.wavss.log,GI01SUMO-SBD12-05-WAVSSA000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/cg_data/dcl12/fdchp/*.fdchp.log,GI01SUMO-SBD12-08-FDCHPA000,recovered_host,Available,
+mi.dataset.driver.fdchp_a.fdchp_a_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00006/instruments/dcl12/FDCHP_sn*/D*/fdchp_*.dat,GI01SUMO-SBD12-08-FDCHPA000,recovered_inst,Expected,Awaiting download
```

Comment:
looks okay

Reply:
The FDCHP recovered instrument is not commented out because you expect to get the data before you run this?